tests/hazmat/backends/test_multibackend.py

```diff
@@ -170,6 +171,11 @@ def load_elliptic_curve_public_numbers(self, numbers):
         if not self.elliptic_curve_supported(numbers.curve):
             raise UnsupportedAlgorithm(_Reasons.UNSUPPORTED_ELLIPTIC_CURVE)
 
+    def elliptic_curve_exchange_algorithm_supported(self):
+        ret = self.exchange
+        self.exchange = not self.exchange
+        return ret
```

Comment:
This is way too confusing. I'd make `exchange_supported` a constructor arg and create two MultiBackend instances